Source/inv.cpp

```diff
@@ -889,7 +889,7 @@ void CheckInvCut(int pnum, Point cursorPosition, bool automaticMove)
 				NewCursor(holdItem._iCurs + CURSOR_FIRSTITEM);
 				if (!IsHardwareCursor()) {
 					// For a hardware cursor, we set the "hot point" to the center of the item instead.
-					SetCursorPos(cursorPosition.x - (cursW / 2), cursorPosition.y - (cursH / 2));
+					SetCursorPos(cursorPosition - (cursSize / 2));

```

Comment:
Hmm, just realised only `Point-Size` is implemented. Is this the only use? If so might as well use an explicit cast/conversion to Displacement like on line 678.